components/Partners/Partner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { s3 } from 'common/constants/urls';
+import styles from './Partner.css';
+
+const Partner = ({ partner }) => (
+  <div className={styles.Partner}>
+    <a href={partner.url} target="_blank" rel="noopener noreferrer">
+      <img className={styles.logo} src={`${s3}${partner.logo}`} alt={partner.name} />
+    </a>
+  </div>
+);
+
+Partner.prototype.propTypes = {
+  partner: PropTypes.shape({
+    logo: PropTypes.string.isRequired,
```

Comment:
Instead of taking an object as the prop, why not take all of these fields and disperse the props at the implementation level?